iterate_dynamodb/README.md

```diff
@@ -0,0 +1,45 @@
+# Use cases for iteration of DynamoDB Data Sources
+
+First of all, you must connect your Tinybird workspace with your Git repository using the [documentation about versions](https://www.tinybird.co/docs/production/working-with-version-control#connect-your-workspace-to-git-from-the-cli) 
+
+### Create a DynamoDB Data Source 
+
+- Create the DynamoDB Connection in your main tinybird workspace using Tinybird CLI, as explain in the [iteration documentation](https://www.tinybird.co/docs/ingest/dynamodb). Connections reside always in the main branch.
```

Comment:
A general note, but there's a mix of lower case-capital case in the README. `Tinybird` and any word referring to a resource should be capital case: Branch, Workspace, Data Source, Pipe, Datafile, etc.

As opposed, a git branch is lowercase.